src/server.js

```diff
@@ -243,7 +243,8 @@ app.post('/search', (req, res) => {
 			.find({
 				$or: [
 					{ title: { $regex: query, $options: 'i' } },
-					{ category: { $regex: query, $options: 'i' } }
+					{ category: { $regex: query, $options: 'i' } },
+					{ metadata: { $regex: query, $options: 'i' } }
```

Comment:
List of query options: https://www.mongodb.com/docs/manual/reference/operator/query/regex/#mongodb-query-op.-options